lib/std/os/linux/sparc64.zig

```diff
@@ -169,7 +169,9 @@ pub extern fn clone(func: fn (arg: usize) callconv(.C) u8, stack: usize, flags:
 
 pub const restore = restore_rt;
 
-pub fn restore_rt() callconv(.Naked) void {
+// Need to use C ABI here instead of naked
+// to prevent an infinite loop when calling rt_sigreturn.
+pub fn restore_rt() callconv(.C) void {
```

Comment:
Honestly I'm not very sure that this is the right fix for it, though it does make the sigaction testcase pass instead of getting trapped in an infinite loop.

Probably needs a review from someone who knows better :eyes:

Reply:
Well, if it's better than status quo, then it can be merged!